app/views/layouts/_tag_edit_items.html.haml

```diff
@@ -2,7 +2,117 @@
+      - when _("Network Port")
+        = n_("%{amount} Network Port Being Tagged", "%{amount} Network Ports Being Tagged", @tagitems.length) % t
+      - when _("Network Router")
+        = n_("%{amount} Network Router Being Tagged", "%{amount} Network Routers Being Tagged", @tagitems.length) % t
+      - when _("Orchestration Stack")
+        = n_("%{amount} Orchestration Stack Being Tagged", "%{amount} Orchestration Stacks Being Tagged", @tagitems.length) % t
+      - when _("Orchestration Template")
+        = n_("%{amount} Orchestration Template Being Tagged", "%{amount} Orchestration Templates Being Tagged", @tagitems.length) % t
+      - when _("PXE Server")
+        = n_("%{amount} PXE Server Being Tagged", "%{amount} PXE Servers Being Tagged", @tagitems.length) % t
+      - when _("Persistent Volume")
+        = n_("%{amount} Persistent Volume Being Tagged", "%{amount} Persistent Volumes Being Tagged", @tagitems.length) % t
+      - when _("Physical Infrastructure Provider")
+        = n_("%{amount} Physical Infrastructure Provider Being Tagged", "%{amount} Physical Infrastructure Providers Being Tagged", @tagitems.length) % t
+      - when _("Physical Server")
+        = n_("%{amount} Physical Server Being Tagged", "%{amount} Physical Servers Being Tagged", @tagitems.length) % t
+      - when _("Playbook (Embedded Ansible)")
+        = n_("%{amount} Playbook (Embedded Ansible) Being Tagged", "%{amount} Playbooks (Embedded Ansible) Being Tagged", @tagitems.length) % t
+      - when _("Repository")
+        = n_("%{amount} Repository Being Tagged", "%{amount} Repositorys Being Tagged", @tagitems.length) % t
+      - when _("Resource Pool")
+        = n_("%{amount} Resource Pool Being Tagged", "%{amount} Resource Pools Being Tagged", @tagitems.length) % t
+      - when _("Security Group")
+        = n_("%{amount} Security Group Being Tagged", "%{amount} Security Groups Being Tagged", @tagitems.length) % t
+      - when _("Service")
+        = n_("%{amount} Service Being Tagged", "%{amount} Services Being Tagged", @tagitems.length) % t
+      - when _("Service Catalog Item")
+        = n_("%{amount} Service Catalog Item Being Tagged", "%{amount} Service Catalog Items Being Tagged", @tagitems.length) % t
+      - when _("Storage Manager")
+        = n_("%{amount} Storage Manager Being Tagged", "%{amount} Storage Managers Being Tagged", @tagitems.length) % t
+      - when _("Switch")
+        = n_("%{amount} Switch Being Tagged", "%{amount} Switches Being Tagged", @tagitems.length) % t
+      - when _("Template (Ansible Tower)")
+        = n_("%{amount} Template (Ansible Tower) Being Tagged", "%{amount} Templates (Ansible Tower) Being Tagged", @tagitems.length) % t
+      - when _("VM or Template")
+        = n_("%{amount} VM or Template Being Tagged", "%{amount} VMs or Templates Being Tagged", @tagitems.length) % t
+      - else
```

Comment:
I added the original string as an else in case I missed some options or new ones are added

Reply:
I also wanted to mention here that if anyone can come up with a better solution than this to resolve the issue please suggest it